Assets/Scripts/Game/WeaponManager.cs

```diff
@@ -481,7 +481,22 @@ private void MeleeDamage(FPSWeapon weapon)
 
                 // TODO: Use correct damage based on weapon and swing type
                 // Just using fudge values during development
```

Comment:
I considered it but it seemed a little difficult, so go ahead! I still need to work on easier things and become more familiar with the project.

I don't think the weapon damage data is being stored anywhere yet. I wasn't sure if it would be okay to just use a source like UESP and manually enter base damage values into the item file, or if the damage data should somehow be extracted from the Daggerfall executable.